CleverCloud/clever-components
# feat(cc-product-card): init component

Fixes #891

## What does this PR do?

This PR introduces the first component for the first step of our new creation tunnel.

## How to review?

* Check the preview, feel free to give your thoughts on the padding, alignment and so on.
  * There's a lot of stories to cover cases that could happen. (Especially with product from the API)
  * If you checkout the branch, you can go to `/sandbox` in your local Storybook. It will let you have a glimpse of the products from the API. You can also customize the product details if you want to play with the card. It's not mandatory though :wink: 
 * Check the code

A review by everyone would be appreciated :slightly_smiling_face: 
The component is quite simple, it should be easy to review :wink:
